### ISSUE TITLE
Wrong stack trace for lambda functions

### ISSUE THREAD

**Issue by schmave** [brython-dev/brython#961]
This is perhaps related to #960 -- when I load this file, z.html:

```html
<html>
<head>

<script type="text/javascript" src="brython/brython.js"></script>
<script type="text/javascript" src="brython/brython_stdlib.js"></script>

<script type="text/javascript">
window.onload = () => {
  brython({ debug: 1 });
};
</script>

<script type="text/python">
import traceback
from browser import window
from z import f
try:
    f()
except:
    window.console.log(traceback.format_exc())
</script>
</head>

<body>
</body>
</html>
```

along with this z.py file:
```python
def g():
    1 / 0

def h():
    x = lambda: g()
    x()

def f():
    h()
```

I see this printed to the JS console:
```
Traceback (most recent call last):
  File "http://localhost:7001/z.html/__main__", line 5, in <module>
    f()
  File "http://localhost:7001/z.py", line 9, in f
    h()
  File "http://localhost:7001/z.py", line 6, in h
    x()
  File "<string>", line 2, in <lambda>
  File "http://localhost:7001/z.py", line 2, in g
    1 / 0
ZeroDivisionError: division by zero
```

I expect to see the lambda identified as being on line 6 of z.py, but it is not.

**Comment by PierreQuentel** [comment]
@schmave 
This bug should also be fixed by the latest commits (060e3244d1a5cae658323ce0af76656921ed00a2 and the next).

**Comment by schmave** [comment]
Yes, it looks good to me. Thanks @PierreQuentel
